docs-src/cloud/temporal-cloud-limits-sheet.md

```diff
@@ -83,6 +83,15 @@ The default maximum number of [Custom Search Attributes](/visibility#custom-sear
 
 For more information on limits on the number and size of custom Search Attributes you can create, see [Custom Search Attributes limits](/visibility#custom-search-attributes).
 
+### What are constraints for Custom Search Attribute in Temporal Cloud?
+
+When creating custom Search Attributes in Temporal Cloud, the attribute names must adhere to the following constraints:
+
+- Maximum characters: 64
+- Allowed characters: `[a-zA-Z0-9.,:-_\/\s]+`
```

Comment:
could we simplify to something like: `a-z`, `A-Z`, `0-9`, `.`, `,`, `:`, `-`, `_`, ` `(space), `(`, `)`, `/`, `\`.

technically, `\s` isn't accurate, since we disallow tabs and newlines (only space character).

here's the regex used in the server side code for reference: 
```go
regexp.MustCompile(`^[a-zA-Z0-9.,:\-_ ()/\\]+$`)
```

Reply:
Awesome. Yeah, good point. Reverted that and used your example.